devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -30,6 +30,7 @@ metadata {
 
 		// eWeLink
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "SA-003-Zigbee", deviceJoinName: "eWeLink SmartPlug (SA-003)", ocfDeviceType: "oic.d.smartplug"
+		fingerprint profileId: "0104", inClusters: "0000,0003,0004,00005,0006", outClusters: "0000", manufacturer: "eWeLink", model: "ZB-SW01", deviceJoinName: "eWeLink 1 Gang Switch"
```

Comment:
Only thing I would recommend is to put spaces in between the clusters so that it is pretty and matches the other surrounding fingerprints.